shell/packages/sandstorm-db/migrations.js

```diff
@@ -305,5 +341,4 @@ function migrateToLatest() {
   }
 }
 
-// Apply all migrations on startup.
-Meteor.startup(migrateToLatest);
+SandstormDb.prototype.migrateToLatest = migrateToLatest;
```

Comment:
The load order in a package is controlled by package.js. See https://github.com/meteor/meteor/issues/2731#issuecomment-58969066